packages/components/src/Text/Text.spec.tsx

```diff
@@ -23,4 +23,24 @@ describe("<Text />", () => {
       </p>
     `);
   });
+
+  it("should pass the passthrough ref", async () => {
+    const TestComponent = () => {
+      const textRef = useRef<HTMLParagraphElement>(null);
+
+      useEffect(() => {
+        textRef.current && textRef.current.focus();
+      });
+
+      return (
+        <Text ref={textRef} size="body" className="passthrough" tabIndex={-1}>
+          Some Text
+        </Text>
+      );
+    };
+    render(<TestComponent />);
```

Comment:
I think it would be simpler and clearer to test using `React.createRef` instead of `React.useRef` (both of which return the same data structure.

Would look something like

```jsx
const textRef = React.createRef();
render(<Test ref={textRef} {...the other props} />);

textRef.current.focus();

const textElement = await screen.getByText("Some Text");
expect(textElement).toEqual(document.activeElement);
```

It's less hoops to jump through, and it's easier to control when the element is focused.

Reply:
+1. I think this suggestion makes the test lightweight enough & worth it

Reply:
Also just remembered there's a `toHaveFocus` matcher coming from jest-dom that we could use here.

Reply:
That's so much better – thank you!

For the `toHaveFocus`, I'm hitting an error that said `expect(...).toHaveFocus is not a function`, which is why I used this roundabout method.